app/templates/todos/index.plush.html

```diff
@@ -18,14 +18,21 @@
                     <tr>
                         <td scope="row" class="task-color"><i class="fas fa-check-circle mr-3 gray-text"></i><%= todo.Title %></td>
                         <td class="limit-date"><%= todo.Deadline.Format("Mon Jan 02") %></td>
-                        <td><a href="<%= editTodoPath({todo_id: todo.ID}) %>"><i class="fas fa-pencil-alt mr-3 gray-text"></i></a><i class="fas fa-trash-alt red-text"></i></td>
+                        <td>
+                            <a href="<%= editTodoPath({todo_id: todo.ID}) %>">
+                                <i class="fas fa-pencil-alt mr-3 gray-text"></i>
+                            </a>
+                            <a type="button" data-toggle="modal" data-target="#deleteTaskModal">
+                                <i class="fas fa-trash-alt red-text"></i>
+                            </a>
+                        </td>
                     </tr>
+                    <%= partial("partials/delete_task_modals.plush.html", {
```

Comment:
<table> should only contain <tr> child elements.
Place this just below the <a data-target="#deleteTaskModal"> element